cluster/ansible/playbooks/roles/master/templates/kube-apiserver.jsonnet

```diff
@@ -0,0 +1,94 @@
+function(cfg)
+  {
+    apiVersion: "v1",
+    kind: "Pod",
+    metadata: {
+      name: "kube-apiserver",
+      namespace: "kube-system",
+      labels: {
+        tier: "control-plane",
+        component: "kube-apiserver",
+      },
+    },
+    spec: {
+      hostNetwork: true,
+      containers: [
+        {
+          name: "kube-apiserver",
+          image: "%(docker_registry)s/kube-apiserver:%(kubernetes_version)s" % cfg.cluster,
+          resources: {
+            requests: {
+              cpu: "250m",
+            },
+          },
+          command: [
+            "/bin/sh",
+            "-c",
+            |||
+              /usr/local/bin/kube-apiserver \
+                --address=127.0.0.1 \
+                --etcd-servers=http://127.0.0.1:4001 \
+                --cloud-provider=gce \
+                --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,ResourceQuota \
+                --service-cluster-ip-range=10.0.0.0/16 \
+                --client-ca-file=/srv/kubernetes/ca.pem \
+                --tls-cert-file=/srv/kubernetes/apiserver.pem \
+                --tls-private-key-file=/srv/kubernetes/apiserver-key.pem \
+                --secure-port=443 \
+                --allow-privileged \
+                --v=4
```

Comment:
If unspecified, --tls-private-key-file is used which is the same key that is passed to the controller-manager